docs/01-app/01-getting-started/16-upgrading.mdx

```diff
@@ -15,15 +15,23 @@ related:
 To update to the latest version of Next.js, you can use the `upgrade` codemod:
 
 ```bash filename="Terminal"
-npx @next/codemod@canary upgrade latest
+npx @next/codemod@latest upgrade latest
 ```
 
 If you prefer to upgrade manually, install the latest Next.js and React versions:
 
-```bash filename="Terminal"
+```bash package="pnpm"
+pnpm i next@latest react@latest react-dom@latest eslint-config-next@latest
+```
+
+```bash package="npm"
 npm i next@latest react@latest react-dom@latest eslint-config-next@latest
 ```
 
+```bash package="yarn"
```

Comment:
`bun` as well

````suggestion
```bash package="yarn"
yarn add next@latest react@latest react-dom@latest eslint-config-next@latest
```

```bash package="bun"
bun add next@latest react@latest react-dom@latest eslint-config-next@latest
```
````

Maybe don't trust the suggestion commit, it is showing a bit odd